### ISSUE TITLE
Fix naming of methods moved from refactored AsyncTask

### ISSUE THREAD

**Issue by bzafiris** [theod0sis/Reactiveness#3]
Currently all extracted and moved methods from an inner AsyncTask class are named after rxXYZ.
In case of multiple async tasks within an activity, this will probably lead to name collisions or overloadings that do not make sense to  the developer.

Extracted methods from refactored async tasks should be named as following:
- doInBackground --> do`TaskName`
- onProgressUpdate --> `taskName`ProgressUpdate
- onPreExecute --> `taskName`PreExecute etc

**Comment by theod0sis** [comment]
The `TaskName`  you refer at the examples is the class name that extends the AsyncTask?

**Comment by bzafiris** [comment]
Right, for inner AsyncTask subclasses, it is the class name. In case of anonymous classes we could use task1, task2 etc, depending on the number of tasks within the Context class.
